rnote-ui/src/appwindow/appwindowactions.rs

```diff
@@ -537,6 +540,27 @@ impl RnAppWindow {
             }),
         );
 
+        // Remove page from doc in fixed size mode
+        action_remove_page_from_doc.connect_activate(
+            clone!(@weak self as appwindow => move |_action_remove_page_from_doc, _target| {
```

Comment:
lets add a record call here:

```rust
let mut widget_flags = canvas.engine().borrow_mut().record(Instant::now());

// ...

appwindow.handle_widget_flags(widget_flags, &canvas);
```

so that the stroke trashing/splitting can be undone. Unfortunately changing document dimensions cannot be undone currently, but that would be a separate improvement.

Reply:
Done